tensorboard/plugins/text/text_plugin.py

```diff
@@ -265,22 +270,25 @@ def _async_index_impl(self):
         'TextPlugin index_impl() thread ending after %0.3f sec', elapsed)
 
   def index_impl(self):
+    run_to_series = self._fetch_run_to_series_from_multiplexer()
     # A previous system of collecting and serving text summaries involved
     # storing the tags of text summaries within tensors.json files. See if we
     # are currently using that system. We do not want to drop support for that
     # use case.
-    run_to_series = collections.defaultdict(list)
     name = 'tensorboard_text'
     run_to_assets = self._multiplexer.PluginAssets(name)
     for run, assets in run_to_assets.items():
       if 'tensors.json' in assets:
         tensors_json = self._multiplexer.RetrievePluginAsset(
             run, name, 'tensors.json')
         tensors = json.loads(tensors_json)
-        run_to_series[run] = tensors
+        run_to_series[run] += tensors
       else:
-        run_to_series[run] = []
+        run_to_series[run] += []
```

Comment:
+= [] on a list has no effect, might as well just omit this whole `else`

Reply:
Done - changed to `= []`. Added a comment.